templates/lynn.html

```diff
@@ -12,21 +12,28 @@
 			<div id="logo">
 				<h1>Chat</h1>
 			</div>
+			<div id="menu">
+				<ul>
+					<li><a href="{{ url_for('login') }}">Logout</a></li>
+				</ul>
+			</div>
 		</div>
 
 		<hr>
 		
 		<div id="container2" class="container">
-			<form>
-				<!--Username-->
-				<label>User</label>
-				<input id="userchat" type="text" name="userchat">	
-				<br><br>
-			
-				<!--Password-->
-				<label>Lynn</label>
-				<input id= "lynnchat" type="text" name="lynnchat">	
-				<br><br>
+			<div class="chat-container">
+				<div id="userchat">
```

Comment:
These text boxes need to extract the text entered by the user and pass it to app.py. Hence just like we extracted values for validating credentials by setting `value = request.form.username` we need to do a similar thing here.  The text entered by the user need to be passed whilst the one coming from app.py.